src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -139,7 +139,17 @@ def _set_artifact_paths(args: argparse.Namespace) -> argparse.Namespace:
     Set paths for all the artifacts.
     """
     if args.artifact_dir == Path(DEFAULT_ARTIFACT_DIR):
-        name = [f"{args.model}"]
+        # Preprocess Huggingface model names that include '/' in their model name.
+        if (args.model is not None) and ("/" in args.model):
```

Comment:
would this be simpler as below?
```
if (args.model is not None):
    filtered_name = re.sub("[/\\\\]", "_", args.model)
```

Reply:
while we're at it, are there other invalid characters?

characters that come to mind `:`, `|`, `>`, `<`, `?`, and all of the non-printable characters.

Reply:
Are regex's ever simpler? 
You raise a good point. I hesitate to go the regex route this close to freeze and without more complete testing.

Reply:
I agree that we should eventually probably sanitize for all characters that are valid in a huggingface model name that aren't valid in triton/genai-perf.

I think starting with just `/` for now is ok, and I think a simple replace is fine:

```
          filtered_name = args.model.replace("/", "_")
```

https://docs.python.org/3/library/stdtypes.html#str.replace